sdk/iOS/OoyalaSkinSDK/ViewControllers/OOSkinViewController.m

```diff
@@ -573,87 +413,20 @@ - (CGRect)videoViewFrame {
 - (void)toggleStereoMode {
   dispatch_async(dispatch_get_main_queue(), ^{
     _isVRStereoMode = !_isVRStereoMode;
-    
-    if (_isVRStereoMode) {
-      [self enterStereoMode];
-    } else {
-      [self exitStereoMode];
-    }
+    self.isVRStereoMode ? [self enterStereoMode] : [self exitStereoMode];
```

Comment:
Please keep the old style of if statement here. It's slightly easier to debug and set a breakpoint for debugging.  Same for the line below.

Reply:
Done

Reply:
Okay